### PR TITLE
Update libs + Fix some errors to build playground

### LINKED ISSUE #214: [nuxt] Display custom error page
Because nuxt (beta version) doesn't display our custom Error page without a page in the project. Maybe it is fix or it will be in the next version.

_Originally posted by @Alketta in https://github.com/locokit/locokit/pull/187#discussion_r1116874871_

### PR DESCRIPTION
**Pull request by Alketta**
Close #214

### CODE REVIEW

**Comment by Alketta** [comment]
@mdartic I'll let you take over the workspace part: some tests are broken and the `configure.test` creation/deletion procedure doesn't work.

File: app/.npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+shamefully-hoist=true
+strict-peer-dependencies=false
```
Comment (mdartic) [inline app/.npmrc:2]: Could you explain the goal of these lines ?

File: app/package-lock.json
Comment (mdartic) [inline app/package-lock.json:1]: I'm not sure this file is useful ?
Why we need a lock file here ?
We are on a workspace architecture, maybe this is a mistake ?

**Review by mdartic — COMMENTED** [review]
Thanks for this PR, 
I'm asking myself some questions, I think you have answers ;-)

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-18
+v18.16.0
```
[the remark this answers is not in the document]
Comment (mdartic) [reply .nvmrc]: Reset to 18

File: api/knexfile.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/first */
-import dotenv from 'dotenv-flow'
-dotenv.config()
+import * as dotenv from 'dotenv'
+
+dotenv.config({ path: process.env.NODE_ENV ? `.env.${process.env.NODE_ENV as string}` : '.env' })
```
[the remark this answers is not in the document]
Comment (mdartic) [reply api/knexfile.ts]: Restore `dotenv-flow` + comment the code with `dotenv` for later discussion

**Pull request merged**